examples/opt/opt_config.py

```diff
@@ -20,8 +23,9 @@
 
 # for server
 engine_server = launch_engine
-tokenizer_path = "facebook/opt-350m"
-# server_host = "127.0.0.1"
+# tokenizer_path = "facebook/opt-125m"
+tokenizer_path = "facebook/opt-30b"
```

Comment:
why use the 30b tokenizer? The 66b tokenizer has bug?

Reply:
Here the config is for 30B model by default. 
I add the 66B tokenizer as the candidate.